app/controllers/application_controller.rb

```diff
@@ -37,7 +37,8 @@ def set_locale
       { name: t('languages.en'), locale: :en },
       { name: t('languages.es'), locale: :es },
       { name: t('languages.ptbr'), locale: :ptbr },
-      { name: t('languages.sv'), locale: :sv }
+      { name: t('languages.sv'), locale: :sv },
+      { name: t('languages.nl'), locale: :nl }
```

Comment:
Sort order? Like other changes below. You generally want these in dictionary order, possibly with english first, but not in history-of-addition order.